onnxruntime/core/providers/cuda/cuda_execution_provider.cc

```diff
@@ -538,6 +538,8 @@ class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCudaExecutionProvider, kOnnxDomain,
 
 class ONNX_OPERATOR_KERNEL_CLASS_NAME(kCudaExecutionProvider, kOnnxDomain, 10, Dropout);
 
+class ONNX_OPERATOR_KERNEL_CLASS_NAME(kCudaExecutionProvider, kOnnxDomain, 11, TopK);
```

Comment:
Hi Randy - Can we also register kernels for TopK opset-9 and opset-10 along with this change ? It seems like by supporting opset-11 TopK, we are inherently supporting the previous opsets too (which have sorted ==1 and largest == 1 always). So, models stamped with opset-9 or opset-10 can get the perf gain too - (#1830)

Reply:
Like this?